File: Practice/src/mypractice/ReverseWords.java
package mypractice;

import java.util.Scanner;

public class ReverseWords {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
        System.out.print("Enter sentence: ");
        String str = sc.nextLine();
        
        String[] words = str.split(" ");
        StringBuilder sb = new StringBuilder();
        for(int i=words.length-1; i>=0; i--) {
        	sb.append(words[i]).append(" ");
        }
        
        System.out.println("Reversed Words: " + sb.toString().trim());
	}

}

//Enter sentence: i am a good girl
//Reversed Words: girl good a am i

File: Preparation_Python/2025/ReverseWords.py
sentence = input("Enter a sentence: ")
words = sentence.split()
reverse_words = ""

for i in range(len(words)-1,-1,-1):
    reverse_words += words[i]
    if i != 0:
        reverse_words += " "
print("Reversed sentence: ", reverse_words)

# Enter a sentence: Hima Bindu
# Reversed sentence:  Bindu Hima